pkg/apis/istio/v1alpha2/istiocontrolplane_types.proto

```diff
@@ -143,20 +141,22 @@ spec:
               value: --livenessProbeInterval=123s
 */
 
-
+// IstioControlPlane is a CustomResourceDefinition (CRD) describing an Istio control plane.
 message IstioControlPlane {
+    // Spec defines the desired state of IstioControlPlane.
     IstioControlPlaneSpec spec = 3;
+    // Status reports the status of the Istio control plane.
     InstallStatus status = 4;
 }
 
 // IstioControlPlaneSpec defines the desired state of IstioControlPlane.
-// The spec is a used to define a customization of the default values that are supplied with an Istio charts release.
+// The spec is a used to define a customization of the default profile values that are supplied with each Istio release.
 // It is grouped at the top level by feature, where behavior of Istio functional areas is specified.
 // Each feature contains components, where k8s resource level defaults can be overridden.
-// Because the spec is a customization API, specifying an empty InstallSpec results in a default Istio controlplane.
+// Because the spec is a customization API, specifying an empty InstallSpec results in a default Istio control plane.
 message IstioControlPlaneSpec {
-    // Prefix added to all namespaces for any installed component.
-    string default_namespace_prefix = 5;
+    // Default namespace if feature or component namespaces are not set.
+    string default_namespace = 5;
```

Comment:
This is going back to the single namespace control plane?

Reply:
No, this is just a default. I think the prefix thing doesn't make sense based on the names I've seen.